### ISSUE TITLE
Navigation justification is broken

### ISSUE THREAD

**Issue by DeoThemes** [WordPress/gutenberg#36402]
### Description

After the recent update 11.9.0 navigation alignment is broken. No matter what alignment option is selected, the CSS variables are undefined

### Step-by-step reproduction instructions

1. Go to Editor (beta)
2. Add navigation block
3. Select justification option

### Screenshots, screen recording, code snippet

![gutenberg_1 19 0_justify_bug](https://user-images.githubusercontent.com/16485360/141290633-c6b9eaf9-3050-4116-8d50-ea977941b8e0.jpg)
 

### Environment info

- WordPress 5.8.2, Gutenberg 11.9.0
- Chrome latest version
- Ona WordPress theme

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by annezazu** [comment]
This seems to be an issue with the Tove theme specifically as I just tested with both TT1 block and Tove:

TT1 Blocks:

https://user-images.githubusercontent.com/26996883/141526194-f294b447-99ce-4b1e-9b3d-471e4f0b8dfd.mov

Tove:

https://user-images.githubusercontent.com/26996883/141526229-e94ddd53-446a-458f-8e9c-2681ff6327d1.mov

I tried testing once more with Tove where there was more space for the navigation block to have but still found the same issue.

cc @andersnoren as a heads up and for your thoughts. Going to close this out for now but am happy to re-open if it's determined to be something with Gutenberg itself over theme implementation.

**Comment by DeoThemes** [comment]
Apparently, it works **only** with TT1 Blocks now, all other themes are broken.

**Comment by annezazu** [comment]
@tellthemachines In digging through the changelog for 11.9, it seems the likely cause is adding in flex layout: https://github.com/WordPress/gutenberg/pull/36169 Would this change require updates on the theme side? Would love your thoughts here!

**Comment by tellthemachines** [comment]
I can't reproduce it when adding a new menu, but it's a known issue for existing menus, and there's a fix waiting for review in #36432.

**Comment by DeoThemes** [comment]
If I remove width: 100%, it works as intended
![navbar_guten_bug](https://user-images.githubusercontent.com/16485360/141886929-69c48baf-f91a-4937-b6d4-4332a23cb8ac.png)

**Fix for Gutenberg 11.9 justify content bug.**
```
@media (min-width: 600px) {
  .wp-block-navigation__responsive-container:not(.hidden-by-default):not(.is-menu-open) {
    width: auto;
  }
}
```

**Comment by tellthemachines** [comment]
Hmm, we need that width set to `100%` otherwise `space-between` justification doesn't work.
